app/views/simulator/edit.html.erb

```diff
@@ -152,7 +152,7 @@
             <li><a class="dropdown-item" href=<%= user_groups_path(current_user) %>>My Groups</a></li>
             <li><a class="dropdown-item" href=<%= notifications_path(current_user) %>>Notifications</a></li>
             <div class="dropdown-divider"></div>
-            <li><a class="dropdown-item" rel="nofollow" data-method="delete" href=<%= destroy_user_session_path %>>Sign Out</a></li>
+            <li><%= button_to "Sign Out", destroy_user_session_path, :method => "delete", class: "dropdown-item signOut-btn" %></li>
```

Comment:
We should fix it with link itself, we may have to check if jquery-ujs is loaded here

Reply:
@tachyons I have tried but not getting it. Can you please help me, "how can I check whether jquery-ujs is loaded or not"?